lib/genevalidator/output.rb

```diff
@@ -238,7 +238,7 @@ def generate_summary_data(overview)
           ['num_predictions', overview[:no_queries]],
           ['num_good_predictions', overview[:good_scores]],
           ['num_bad_predictions', overview[:bad_scores]],
-          ['num_predictions_with_insufficient_evidence', overview[:nee]],
+          ['num_predictions_with_insufficient_evidence', overview[:insufficient_BLAST_hits]],
```

Comment:
Shouldn't the key be changed to `num_hits_with_insufficient_blast_hits`?

Reply:
`num_predictions_with_insufficient_blast_hits`

Reply:
>

Reply:
Right